packages/ic-representation-independent-hash/src/representation_independent_hash.rs

```diff
@@ -96,4 +98,18 @@ mod tests {
 
         assert_eq!(result, reordered_result);
     }
+
+    #[test]
+    fn hash_bytes() {
+        let map: Vec<(String, Value)> = vec![
+            ("bytes".into(), Value::Bytes(vec![0x01, 0x02, 0x03, 0x04])),
+        ];
+        let expected_hash =
+            hex::decode("546729666d96a712bd94f902a0388e33f9a19a335c35bc3d95b0221a4a574455")
```

Comment:
I checked this hash against the JS implementation of the representation independent hash.